addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -320,6 +375,34 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         this.trigger_up('open_move_to_modal', {});
     },
 
+    /**
+     * Update the btn-chatter appearance and hide the chatter section in the
+     * view. Closes the chatter.
+     *
+     * @private
+     */
+    _onCloseChatter() {
+        this.el.querySelector('.o_knowledge_chatter').classList.add('d-none');
+        this.el.querySelector('.btn-chatter').classList.remove('active');
+        this._closeChatter();
+    },
+
+    /**
+     * Update tbe btn-chatter appearance and show the chatter section in the
+     * view. Open the chatter.
+     *
+     * @private
+     * @param {OdooEvent} ev
+     */
+    async _onOpenChatter(ev) {
+        ev.stopPropagation();
+        if (this.state.res_id) {
+            this.el.querySelector('.o_knowledge_chatter').classList.remove('d-none');
```

Comment:
By the way, why not using show and hide ? Any reason (maybe due to styling instead of classes) ?

Reply:
Are you talking about JQUERY show() method ? (I'm trying to avoid JQUERY if possible, following JS-team incentive)

The initial state is a `d-none` class in the xml file, so I decided to manipulate it like this, but if you think there is a better way I can change this^^